ykrt/src/jitstats.rs

```diff
@@ -17,7 +14,14 @@ use std::{
 };
 use strum::{Display, EnumCount, EnumIter, IntoEnumIterator};
 
+/// Record yk statistics if enabled. In non-testing mode this is only enabled if the end user
```

Comment:
comma after `mode`?

Reply:
I think it's fine with or without?

Reply:
Up to you, but I was jarred on first read.

Reply:
thanks